docs/reporting/key-concepts.mdx

```diff
@@ -0,0 +1,66 @@
+
+Reports are designed by report authors (administrators) using a web-based tool called Geocortex Report Designer. They are saved to the Geocortex Reporting service, with an item in the content area of the author's ArcGIS account. To allow users to run a report, the author shares the ArcGIS item with them. End users use the reports designed by the author to create on-demand, print-ready reports about maps and their features.
+
+End users run reports from web applications that the author has configured to host the reports (host applications). Alternatively, a report can be run using the Geocortex Reporting REST API. The output report is a PDF.
+
+## Geocortex Report Designer
+
+[Geocortex Report Designer](https://apps.geocortex.com/reporting/designer/) provides an extensive suite of tools for laying out reports, specifying the data that you want to include, and presenting the data using charts, tables, and a wide variety of other configurable elements. Organizations can sign into the cloud-hosted report designer, or they can host it themselves on their own web server. They can then author as many report templates as their application requirements demand and save them into ArcGIS Online or Portal for ArcGIS as items. Sharing reports with end users allows them to create on-demand, print-ready PDF reports about maps and features, using data from the user's current session.
+
+Report Designer has a built-in Preview environment for running reports. Using the Preview is the quickest way to test a report while you are designing it.
+
+## Geocortex Reporting Service
+
+The [Geocortex Reporting REST API](https://apps.geocortex.com/reporting/service/api-viewer/index.html?url=../swagger.json#/) can be used to asynchronously run a report, watch for report completion and download the completed report as a PDF.
+
+:::note
+Prior to running a report, you must create a new report template using [Geocortex Report Designer](https://apps.geocortex.com/reporting/designer/).
+:::
+
+The basic steps for running a report are:
+
+1.  Obtain a run token from the [/auth/token/run](https://apps.geocortex.com/reporting/service/api-viewer/index.html?url=../swagger.json#/auth/post_auth_token_run) endpoint.
+    -   Only required if the report is secured or accesses secured content
+2.  Run the report with a request to [/job/run](https://apps.geocortex.com/reporting/service/api-viewer/index.html?url=../swagger.json#/job/post_job_run).
+
+    -   The request body requires a template object which consists of an ArcGIS Portal url and itemid. An optional array of parameters may also be passed to the report.
+
+    <!-- cspell:disable -->
+
+        ```json
+        {
+            "template": {
+                "itemid": "5a810f3ac13454d6bfd2ac22ed0cc4b7",
+                "portalUrl": "https://arcgis.com"
+            },
+            "parameters": [...]
```

Comment:
Maybe an example would be helpful here? Something like (will likely need prettier formatting):

```suggestion
            "parameters": [
            	{
                    "name": "SomeParam",
                    "containsMultipleValues": true,
                    "values": [1, 2]
                },
                {
                    "name": "AnotherParam", 
                    "containsMultipleValues": false, 
                    "value": 42
                }
            ]
```